tokio-process/src/lib.rs

```diff
@@ -394,7 +407,10 @@ impl<T: Kill> ChildDropGuard<T> {
     }
 }
 
-impl<T: Kill> Kill for ChildDropGuard<T> {
+impl<T: Kill> Kill for ChildDropGuard<T>
+where
+    T: Unpin,
```

Comment:
Doesn't seem like we actually need `Unpin` here? I think that also extends such that we can remove this bound from the drop impl and the declaration as well